source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL, getRepoGQL} from '../github-helpers';
+
+const getReleaseID = async (): Promise<string> => {
+	const tagName = location.pathname.split('/').pop()!;
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			release(tagName: "${tagName}") {
+				id
+			}
+		}
+	`);
+
+	// => atob(repository.release.id) => "07:Release27300592"
+	return atob(repository.release.id).replace(/.*Release/, '');
+};
+
+async function convertToDraft({delegateTarget: draftButton}: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	draftButton.textContent = 'Converting...';
+	draftButton.append(<LoadingIcon className="ml-2" width={16}/>);
+
+	const releaseID = await getReleaseID();
+	const response = await api.v3(`repos/${getRepoURL()}/releases/${releaseID}`, {
+		method: 'PATCH',
+		body: {
+			draft: true
+		}
+	});
+
+	if (!response.ok) {
+		draftButton.textContent = 'Convert Failed! See console for details';
+		return;
+	}
+
+	location.href = new GitHubURL(response.html_url).assign({branch: 'edit'}).href;
+}
+
+function init(): void | false {
+	const editButton = select('.BtnGroup a[href*="releases/edit"]')!;
+	if (!editButton || select.exists('.label-draft')) {
+		return false;
+	}
+
+	editButton.after(
+		<a
+			className="btn BtnGroup-item text-orange rgh-convert-draft"
+		>
+			Convert to draft
+		</a>);
+
+	delegate(document, '.rgh-convert-draft', 'click', convertToDraft);
+}
+
+void features.add({
+	id: __filebasename,
+	description: 'Adds a button next to a single release to cover it to a draft.',
+	screenshot: 'https://user-images.githubusercontent.com/16872793/89732990-767e0380-da20-11ea-8ac5-b617701eeb29.png'
+}, {
+	include: [
+		pageDetect.isSingleTagPage
```

Comment:
Why was this named “page”? All of them are pages. This should just be `isSingleTag`. Can you send a PR there with the usual rename+deprecation?

Reply:
Yes. But not until tomorrow for sure. 
I am swamped at work and I am trying to finish 5 pr's LOL

Reply:
And `isActionPage`:

https://github.com/fregante/github-url-detection/blob/7ab40fbe7e5f6f2a9cf163ef7f1aaa2b69ab59a9/index.ts#L438-L443

Reply:
https://github.com/fregante/github-url-detection/pull/32